.circleci/config.yml

```diff
@@ -77,3 +182,41 @@ jobs:
       - store_artifacts:
           path: test-reports
           destination: test-reports
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - static-check
+      - install-all-and-check-requirement
+      - pylint:
+          requires:
+            - install-all-and-check-requirement
+      - pre-test:
+          name: pre-test 3.5.5
```

Comment:
I don't know if it is available, but we should prefer to use 3.5.3, as that is our minimum supported version.

Reply:
circle don't have 3.5.3-stretch, only jessie, it will fail our deps installation

Reply:
ok, let's stick with 3.5.5